query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/Params.java

```diff
@@ -56,6 +56,58 @@ private Params(
     return bytesStringParams;
   }
 
+  @Override
+  public String toString() {
```

Comment:
Should be easy to unit test these, no?

Reply:
It's only used for logging, hence didn't bother to add the unit test. Equals is unit tested though.